src/labels/CustomLabels.labels

```diff
@@ -7655,6 +7663,110 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
         <shortDescription>Gift Processing configuration error</shortDescription>
         <value>Your org is missing required configuration for Gift Processing. Contact your admin for help.</value>
     </labels>
+    <labels>
+        <fullName>gsApplicationStatus</fullName>
+        <categories>Getting Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Application Status label for the application status component</shortDescription>
+        <value>Application Status: </value>
+    </labels>
+    <labels>
+        <fullName>gsApplicationStatusModalHeader</fullName>
+        <categories>Getting Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Header of the modal label for the application status component</shortDescription>
+        <value>Check Application Status</value>
+    </labels>
+    <labels>
+        <fullName>gsApplicationUrl</fullName>
+        <categories>Getting Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Url of power of us</shortDescription>
+        <value>https://www.salesforce.org/power-of-us/</value>
```

Comment:
I was thinking you'd move the email to the html. I recommend not putting URL's and email addresses in custom labels. This has been called out as security issue because a malicious user  could modify a URL (by using the Overrride option in the org) so that any other users that click that link redirect them to a website that does something the user did not intend it to do. In other areas where we've hardcoded URL's (primarily in older visualforce pages), we actually push those through an allowlist component that strips out unsupported URL's. For an LWC though, we should just put the URL into the HTML. It's easily updated in either location.

Reply:
Agreed, we really should not use Custom Labels for anything other than a translatable string.